### ISSUE TITLE
All In Game Items Reset For No Reason

### ISSUE THREAD

**Issue by Weymere** [Wynncraft/Issues#2013]
After being an avid Wynncraft player for days in 2013-2014, I decided to take a break from the game for a bit and started doing other things. Today, however, when I came back to the game, all of my in game items were reset, but I  my levels and quest progress. This means that I am a level 15-20 class in the starter area, with no oway to get back to my position by doing quests. So I ask of you that you would please reset my characters progress,but a different solution would be appreciated. My IGN is Bald_Butter, although it used to be GoldenFireHunter. Thank you! Also, all I have is a starter bow/ wand, like you would if you were just starting, but I did keep my levels and quest progress

**Comment by Weymere** [comment]
Also, all I have is a starter bow/ wand, like you would if you were just starting, but I did keep my levels and quest progress

**Comment by Trixter313** [comment]
So, are you saying that even important quest items are gone from your inventory too?

**Comment by Weymere** [comment]
Yes

**Comment by Trixter313** [comment]
Try typing /fixquests into chat. It's experimental but seems to be the only solution. Staff members rarely reset an account on request but you never know. I really hope this command fixes your issue. Good luck and report back!

**Comment by Weymere** [comment]
Unfortunately, that does not seem to have fixed my issue D:.... I assume my only option is to ask for a reset from staff. Thanks for the suggestion though.

**Comment by Trixter313** [comment]
Ok... I'll continue thinking of possible fixes. This is the official bug reporter so staff members check here periodically. You might be able to get a faster response by emailing support@wynncraft.com. I hope this gets fixed soon. :)

**Comment by Weymere** [comment]
Okay, thanks for the help! I only held off emailing them, because I hoped for a solution here, but I guess I'll email them as well. I appreciate your advice though :hamburger:

**Comment by Weymere** [comment]
As another note, I know as a fact that I did not die, or run out of soul points, so that cause of this problem is not responsible.

**Comment by Trixter313** [comment]
Yeah, when you run out of soul points and die you at least keep your quest items. That's the main issue I see here, no quest items.

**Comment by Weymere** [comment]
Yeah, I have no idea :P

**Comment by AckroWynn** [comment]
The /fixquests command returns all of your lost quest items. Essentially, items with the tag "Quest Item".

**Comment by Trixter313** [comment]
@Weymere tried the /fixquests command and reported back that it didn't work for them. Is there anything else that can be done for them?

**Comment by iColoringBook** [comment]
This has also happened to me. I was in detlas near the item identifier. The server was restarting and as I joined back, I joined with an empty inventory. I logged in slightly laggy and it usually starts with my inventory empty then gets everything loaded but that time, I kept the empty inventory.I had lost everything from equipped armor to emeralds and quest items. This has also happened to me a while back before then so that would be the second time.
